packages/react/src/components/Drawer/Drawer.test.tsx

```diff
@@ -305,7 +305,7 @@ test('should not trap focus when behavior is non-modal', async () => {
     'aria-hidden',
     'true'
   );
-  await user.keyboard('{Tab}');
+  document.body.focus();
   expect(document.body).toHaveFocus();
```

Comment:
This reads kinda funny:
- focus the body
- assert that the body is focused

Is the expect necessary? Just to confirm that body.focus() actually worked?

Reply:
Yeah I guess the assertion is a bit redundant since we should assume that the body has focus after we call `body.focus()`.